config/routes.rb

```diff
@@ -8,6 +8,9 @@
     post "datatable", on: :collection
   end
 
+  get "table_state", to: "volunteers#table_state"
```

Comment:
this should be inside of the volunteers resource (https://github.com/rubyforgood/casa/pull/4669/files#diff-959bc9abc46a55332bb64d5155a79323afa75a50ec1a2137ddd22d926f62c6c5L116)

Reply:
I had issues with this one too. This is old code ant it seem like it messes with other routes when I nest it. I'll need help on this one.